python/sglang/srt/openai_api/adapter.py

```diff
@@ -1213,7 +1213,11 @@ def v1_chat_generate_request(
         top_logprobs_num=top_logprobs_nums,
         stream=all_requests[0].stream,
         return_text_in_logprobs=True,
-        rid=request_ids,
+        rid=(
```

Comment:
I can't reproduce this line. I understand the issue might come from list type not hashable. Can we trace the origin of the issue? As I feel we should fix it in normalize request fields instead of here.